content/articles/matlab-graphical-user-interface/index.md

```diff
@@ -0,0 +1,107 @@
+Let's look at each of the components before using them.
+1. `Push Button` – It has a raised background and it performs the function of calling the callback function for the execution of different programs.
+2. `Slider` - It is similar to the scroll bar and continuously changes numerical value. The callback function is called and executed when the user changes the pad position. The pad position can be changed by dragging the pad to the required position or click the forward or backward arrow but for this, it changes by a predetermined value.                                  
+3. `Radio Button` – It has the `on` and `off` states. Its state can be changed by clicking on it. When there is a solid circle inside the hollow circle then it is `on` but when the hallowed circle is empty, then it is `off`. They are grouped and only one can be `on` at a time.
+4. `Check Box` – It has the `on` and `off` state. The `on` state is represented by a `tick` inside the box. They are grouped and several checkboxes can be `on` at the same time.
+5. `Edit Text` – It is for getting string input from the user. The input is then modified to numbers using `str2num` or `str2double` function. 
+6. `Static Text` – It is for adding labels that remain unchanged on the GUI and has no callbacks.
+7. `Pop-up Menu` – It is for listing selections but only the current selection is visible. You click on it to view the selections and the callback function is executed depending on the selection.
+8. `List box` – It shows a list of selections just as the pop-up menu. In the list box, all the selections are visible throughout unlike the pop-up menu where only the current selection is visible.
+9. `Toggle Button` – It functions similarly to the radio button but different appearance in the GUI. It's the appearance that makes the difference. 
+10. `Table` – It is for the addition of a spreadsheet to the GUI. In case of modification of the input, the callback is executed.
+11. `Axes` – They are for the addition of images, charts, and plots to the GUI. They have no callback function.
+12. `Panel` – It is for grouping of several components and named according to the functions of components. It doesn't have a callback function. We should first add the pannel before inputting the components into it.
+13. `Button Group` – It is just like the pannel but used to group radio buttons. When we group the radio buttons here, several can be `on` at the same time but this is corrected by writing the callback functions. Just as the panel, we first add the button group before adding the radio buttons.
+### Example GUI
+We want to create a simple GUI. We first add the required components which are, Panel, Static text, Axes, Push Button, Slider, and Edit Text. Add the components as explained earlier and arrange them as shown below;
```

Comment:
`Panel, Static text, Axes, Push Button, Slider, and Edit Text`

Put each of these components within ` `` `, since they are more like keywords.